Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN ./manage.py collectstatic --noinput
 RUN apt-get clean && apt-get purge
 USER mitodl
 
+RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
```

Comment:
Yep moving `USER mitodl` after that line makes the build pass for me. It shouldn't build correctly as-is, the `mitodl` shouldn't have permission to modify `/var` right?